admin/src/components/link.tsx

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/react';
+
+import { openInBrowser } from '../utils';
+
+type LinkProps = {
+	href?: string;
+	onClick?: () => void;
+	children: React.ReactNode;
+};
+
+export function Link({ href, onClick, children }: LinkProps) {
```

Comment:
I might just be blind but I can't see this component being used anywhere? Potentially for link buttons we could use Atlaskits' [link button](https://atlassian.design/components/button/examples#link), or `subtle-link` depending on the use case.

Reply:
oof - bad commit splitting on my part. This got added in because I was rendering a link for users to sign into a different account. But I changed that into a TODO to keep this PR small since it required updating the `/admin/checkAuth` route. FWIW it's used in the subsequent PR. I'll delete this and just update the subsequent PR to use link button instead!